cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -547,20 +545,25 @@ def resolve_job_url(self):
         # what if scw list from request overwrites that in self.par_dict?
         self.set_scws_related_params(request)
 
-        self.validate_job_id(request_parameters_from_scratch_dir=True)
+        self.validate_job_id(request_parameters_from_scratch_dir=True, job_resolution=True)
 
         return self.generate_products_url_from_par_dict(
             self.app.config['conf'].products_url,
             self.par_dic
             )
 
-    def validate_job_id(self, request_parameters_from_scratch_dir=False):
+    def validate_job_id(self, request_parameters_from_scratch_dir=False, job_resolution=False):
```

Comment:
job_resolution is no more used; maybe it can be kept, but with TODO or a message

Reply:
I will just remove it. It is no longer needed